src/diffusers/models/modeling_utils.py

```diff
@@ -1404,6 +1405,54 @@ def float(self, *args):
         else:
             return super().float(*args)
 
+    def compile_repeated_blocks(self, *args, **kwargs):
+        """
+        Compiles *only* the frequently repeated sub-modules of a model (e.g. the
+        Transformer layers) instead of compiling the entire model.  This
+        technique—often called **regional compilation** (see the PyTorch recipe
+        https://docs.pytorch.org/tutorials/recipes/regional_compilation.html)
+        can reduce end-to-end compile time substantially, while preserving the
+        runtime speed-ups you would expect from a full `torch.compile`.
+
+        The set of sub-modules to compile is discovered in one of two ways:
+
+        1. **`_repeated_blocks`** – Preferred.  Define this attribute on your
+        subclass as a list/tuple of class names (strings).  Every module whose
+        class name matches will be compiled.
+
+        2. **`_no_split_modules`** – Fallback.  If the preferred attribute is
```

Comment:
`_no_split_modules` is for specifying the modules that have residual connections so that we don't split them when using `device_map` on multiple devices. I don't think it's legacy yet.

Reply:
I don't think we need fallback/proxity here
we can make sure every model we have this attribute specified :)